test/utils/Lib.sol

```diff
@@ -218,4 +218,9 @@ library Lib {
         return
             keccak256(abi.encodePacked(b1)) != keccak256(abi.encodePacked(b2));
     }
+
+    function eq(string memory b1, string memory b2) public pure returns (bool) {
+        return
+            keccak256(abi.encodePacked(b1)) == keccak256(abi.encodePacked(b2));
```

Comment:
```suggestion
        if(bytes(b1).length != bytes(b2).length) {
            return false;
        } else {
            return keccak256(b1) == keccak256(b2);
        }
```

Kinda nitty but [adding the length check comes with some gas savings at little to no downside](https://fravoll.github.io/solidity-patterns/string_equality_comparison.html)